symfony.lock

```diff
@@ -222,17 +225,14 @@
         "version": "1.0.0-alpha7"
     },
     "drush/drush": {
-        "version": "10.5.0"
+        "version": "10.3.6"
```

Comment:
Downgrade?

Reply:
@cafuego Not really my doing, when the php requirement is set to PHP 8.0, composer downgrades drush...